translations/data/main.json

```
@@ -1,13 +1,15 @@
 {
     "site_title": {
         "en-us": "Join the fediverse!",
+	"de-de": "Komm ins Fediverse !",

```

An extra space again.

---

Extra spaces between a word and a following punctuation are not used in german language.
```suggestion
	"de-de": "Komm ins Fediverse!",

```